components/levoit/levoit.cpp

```diff
@@ -444,7 +444,19 @@ void Levoit::handle_payload_(LevoitPayloadType type, uint8_t *payload, size_t le
       pm25NAN = (payload[12] == 0xFF && payload[13] == 0xFF);
       if (!pm25NAN) {
         uint16_t raw_value = (payload[13] << 8) + payload[12];
-        uint32_t new_pm25Value = (raw_value * 10) / 10000;
+        
+        switch (this->device_model_) {
+          case LevoitDeviceModel::CORE_200S:
+            new_pm25Value = raw_value / 10000;
```

Comment:
```suggestion
            new_pm25Value = raw_value / 1000;
```
The old value was `10 times raw_value divided by 10000` which means `1000` and not `10000`. :)